modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc

```diff
@@ -10,21 +10,87 @@ image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", ti
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
 
-When you deploy an application to CloudHub, you have access to the validated OS updates, security patches, and bug fixes that MuleSoft provides regularly.
 
-You can keep your CloudHub applications up to date by manually updating them at your convenience.
-If you don't update apps manually, MuleSoft applies the updates automatically.
+To ensure the security and stability of CloudHub, MuleSoft releases security updates
+and monthly patches and applies them automatically on a schedule.
 
-When a new runtime version is available for your CloudHub application, the *Applications* page displays a banner and the *Update Available* tab lists the apps that require updates:
+Automatic updates use blue-green deployment, which ensures zero downtime.
 
+== Security Updates 
+
+MuleSoft regularly runs scans to identify security vulnerabilities in Mule runtime engine, JVM,
+and the underlying operating system, and then automatically applies security patches based on the following Salesforce SLA:
+
+[%header,cols="20,25,40"]
+|===
+|Severity Level | Severity Definition | Patch Applied Within 
+|P0 | Critical | 7 days
+|P1 | High | 30 days
+|===
+
+To minimize disruption, the monthly date patch update always includes patches for any P1 security issues.
+P0 security patches might be included in the monthly update or released separately, depending on the SLA requirements.
+
+== Monthly Date Patch Updates 
+
+Once per month, MuleSoft releases a _date patch_ update, 
+which includes backward-compatible bug fixes to address any issues discovered in Mule runtime engine.
+The monthly date patch update might also include security patches.
+
+The format of the version number for these updates is:
+
+_Major_._Minor_._Patch_ - _Date_ 
+
+Here's an example version number, including the patch date:
+
+`4.2.2 - 05-04-2020`
+
+=== Update Schedule 
+
+The schedule for date patch updates is:
+
+* First Monday of the month, MuleSoft releases the patch.
++
+You can apply the update manually or let MuleSoft apply the patch automatically.
++
+See <<Update the Runtime Version Manually>>
+* Third Saturday of the month, MuleSoft automatically applies the patch.
++
+When an automatic update occurs, the audit log for your app contains an entry by user `Anypoint Staff`,
+which shows the date and time an update occurred and whether it was successful.
++
+If the update is successful, no action is required.
+See <<Zero-Downtime Restart Limitations>>.
+
+For the schedule of upcoming releases, see <link that has the dates>.
+
+[NOTE]
+You must have an Enterprise support account to view the release schedule.
+
```

Comment:
We can introduce the option to take these options themselves here.
"If the 3rd week of the month is not suitable or your application requires manual updates, please see the <Manual Update section>"

Reply:
Oops, didn't see this before. Will add.